README.md

```diff
@@ -45,8 +45,7 @@ Examples of UPP products include:
 
 
 ## User Support
-Support for the UFS UPP is provided through the [UFS Forum](https://forums.ufscommunity.org/)
-by the Developmental Testbed Center (DTC). 
+Support for the UFS UPP is provided through [GitHub Discussions](https://github.com/NOAA-EMC/UPP/discussions).
```

Comment:
I suggest instead of discussion, use issues.

Reply:
@edwardhartnett - Thanks for the suggestion, I can see the advantages to that. I've also seen Discussions used effectively for engagement with community too. It's mostly up to EPIC since they are now the ones providing community UFS support for UPP. So perhaps something they want to consider or discuss with with the larger UPP code management team to decide where/how they want the community to engage.